site/default/templates/layout.mustache

```diff
@@ -61,15 +61,15 @@
 	{{^hideFooter}}
 	<footer>
 		<nav class="container">
-			<a class="brand javascriptmvc" href="index.html"></a>
+			<a class="brand donejs" href="index.html"></a>
```

Comment:
Are there any styles or scripts that use this class? If not, can it be removed?

Reply:
There are none. I'm not opposed to semantic classes that are unused though. Either way.